### ISSUE TITLE
Heading order reversed in `tart list` ?

### ISSUE THREAD

**Issue by dan-snelson** [cirruslabs/tart#88]
First, five words: "**Shine on, you crazy diamonds!**"

When using `tart list`, the headings seem reversed to me:

```
% brew info tart
cirruslabs/cli/tart: stable 0.4.0

% tart list
Name	Source
local	12.4_21F79-base
local	12.4_21F79-snapshot-2022-05-18-115200
local	12.4_21F79-snapshot-2022-05-18-115908
```

**Comment by dan-snelson** [comment]
Replicated same unexpected behavior after upgrading to `0.4.1`

```
% brew info tart
cirruslabs/cli/tart: stable 0.4.1

% tart list     
Name	Source
local	12.4_21F79-base
local	12.4_21F79-snapshot-2022-05-18-115200
local	12.4_21F79-snapshot-2022-05-18-115908
```

**Comment by dan-snelson** [comment]
Confirmed fixed in `0.5.0`; thanks for the _speedy_ turnaround.
